docs/upgrade.html

```diff
@@ -21,6 +21,13 @@
 
 <h5><a id="upgrade_300_notable" href="#upgrade_300_notable">Notable changes in 3.0.0</a></h5>
 <ul>
+    <li>ZooKeeper has been upgraded to 3.6.3, and that version has a hard dependency on the
+        <code>io.dropwizard.metrics:metrics-core:3.2.5</code> library due to the new metrics subsystem added in 3.6.0.
+        Setting <code>metricsProvider.className=org.apache.zookeeper.metrics.impl.NullMetricsProvider</code> in your
+        zookeeper.properties file does not remove this dependency.  ZooKeeper will fail to start with
+        <code>java.lang.NoClassDefFoundError: com/codahale/metrics/Reservoir</code> if you do not have the above library
+        on your CLASSPATH.
+    </li>
```

Comment:
Isn't this handled automatically for users?

Reply:
> Isn't this handled automatically for users

Yeah, good point, no need to discuss it here.  Will remove.